apps/api/src/domain/models/digital-text/entities/digital-text.from-event-history.spec.ts

```diff
@@ -191,6 +230,34 @@ describe(`DigitalText.fromEventHistory`, () => {
             });
         });
 
+        describe(`when the title has been translated`, () => {
+            const eventStream = [digitalTextCreated, digitalTextTitleTranslated];
+
+            const digitalTextBuildResult = DigitalText.fromEventHistory(eventStream, id);
+
+            expect(digitalTextBuildResult).toBeInstanceOf(DigitalText);
+
+            const digitalText = digitalTextBuildResult as DigitalText;
+
+            const titleTranslationSearchResult = digitalText.title.getTranslation(
+                languageCodeForTitleTranslation
+            );
+
+            expect(titleTranslationSearchResult).toBeInstanceOf(MultilingualTextItem);
+
+            const titleTransation = titleTranslationSearchResult as MultilingualTextItem;
+
+            expect(titleTransation.text).toBe(titleTranslationText);
+        });
+
+        describe(`when audio has been added to the title`, () => {
+            it(`should return audio for the title`, () => {
+                const eventStream = [digitalTextCreated, addAudioForDigitalTextTitle];
+
+                const digitalTextBuildResult = DigitalText.fromEventHistory(eventStream, id);
```

Comment:
You need to 
- check that `digitalTextBuildResult` is an instance of DigitalText
- create a new const `updatedDigitalText` set to `digitalTextBuildResult` but cast as `DigitalText`
- call `updatedDigitalText.getAudioForTitleInLanguage` using the languageCode that was on the event payload (I think you want to target the original text item) and store the result to `audioIdSearchResult`
    - assert (expect) that `audioIdSearchResult` is not an instance of `InternalError`
    - assert (expect) that `audioIdSearchResult` is equal to the audio ID you put on the event payload

Reply:
done